src/sentry/models/releases/set_commits.py

```diff
@@ -58,19 +57,14 @@ def set_commits(release, commit_list):
         # the same release rapidly for different projects.
         raise ReleaseCommitError
 
-    if features.has("organizations:set-commits-updated", release.organization):
-        create_repositories(commit_list, release)
-        create_commit_authors(commit_list, release)
+    create_repositories(commit_list, release)
+    create_commit_authors(commit_list, release)
```

Comment:
Are you sure we don't still want these inside of the lock, but not the transaction?

Reply:
added inside of lock, but not transaction.